src/cockpit/internal_endpoints.py

```diff
@@ -136,6 +136,7 @@ class cockpit_User(bus.Object):
     name = bus.Interface.Property('s', value='')
     full = bus.Interface.Property('s', value='')
     id = bus.Interface.Property('i', value=0)
+    gid = bus.Interface.Property('i', value=0)
```

Comment:
Honestly, this isn't the thing we need from cockpit-files.

What we actually want is the name of the primary group.

Reply:
I am aware, but the gid is kinda free to add